jpegencoderhelper.cpp

```diff
@@ -142,46 +142,58 @@ void JpegEncoderHelper::setJpegCompressStruct(int width, int height, int quality
 
 bool JpegEncoderHelper::compressYuv(jpeg_compress_struct* cinfo, const uint8_t* yBuffer,
                                     const uint8_t* uvBuffer, int lumaStride, int chromaStride) {
+    size_t u_plane_size = chromaStride * cinfo->image_height / 2;
+    uint8_t* y_plane = const_cast<uint8_t*>(yBuffer);
+    uint8_t* u_plane = const_cast<uint8_t*>(uvBuffer);
+    uint8_t* v_plane = const_cast<uint8_t*>(u_plane + u_plane_size);
+
+    const int aligned_width = ALIGNM(cinfo->image_width, kCompressBatchSize);
```

Comment:
Nit: maybe renaming it to `aligned_luma_width` to be more clear.

Reply:
Updated patchset to resolve this.